src/galax/potential/_potential/base.py

```diff
@@ -363,7 +382,83 @@ def integrate_orbit(
         -------
         orbit : Orbit
             The integrated orbit evaluated at the given times.
+
+        See Also
+        --------
+        :meth:`~galax.potential.AbstractPotentialBase.compute_orbit`
+            A higher-level function that computes the orbit using time
+            information from `w0`.
+        galax.dynamics.compute_orbit
+            The function which
+            :meth:`~galax.potential.AbstractPotentialBase.compute_orbit` calls.
+        galax.dynamics.integrate_orbit
+            The function for which this method is a wrapper. It has more details
+            and examples.
         """
         from galax.dynamics._dynamics.orbit import integrate_orbit
 
         return cast("Orbit", integrate_orbit(self, w0, t, integrator=integrator))
+
+    def compute_orbit(
+        self,
+        w0: PhaseSpacePosition | PhaseSpaceTimePosition | BatchVec6,
+        t: VecTime | Quantity,
+        *,
+        integrator: "Integrator | None" = None,
+    ) -> "Orbit":
+        """Compute an orbit in a potential.
+
```

Comment:
Add a note up here about the difference with `integrate_orbit()`?
```suggestion

This method is similar to :meth:`~galax.potential.AbstractPotentialBase.integrate_orbit()`, but can behave differently when a :class:`~galax.dynamics.PhaseSpacePositionTime` is passed in. ...explain...

```